index.html

```diff
@@ -12,12 +12,25 @@
       <div id="first">
         <div class="circle"></div>
       </div>
-      <div id="second"></div>
-      <div id="third"></div>
+
+      <div id="second">
+        <img
+          src="https://images.pexels.com/photos/577585/pexels-photo-577585.jpeg?auto=compress&cs=tinysrgb&w=1260&h=750&dpr=1"
+          alt=""
+        />
+      </div>
+
+      <div id="third">
+        <iframe
+          width="900"
```

Comment:
Please place all style related attributes in the style.css file accordingly. Create more css classes or Ids if necessary for each element.

Reply:
This is part of the embedded code from youtube video so i suggest we leave it as it is.

Reply:
Ok then I aggree we can leave it for now